### PR TITLE
docs: translate `README.md` into Japanese

### LINKED ISSUE #1125: 📎 Translate README.md to Japanese
### Description

Similar to zh-CN, please translate it into Japanese.

ref: https://github.com/biomejs/biome/blob/main/packages/%40biomejs/biome/README.zh-CN.md

### PR DESCRIPTION
**Pull request by K-tecchan**
## Summary
Translate readme.md to japanese

close #1125

### CODE REVIEW

File: packages/@biomejs/biome/README.ja.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+</div>
+
+**Biome** はウェブプロジェクトのための高性能なツールチェーンであり、プロジェクトの健全性を維持するための開発者ツールを提供することを目的としています。
+
+**Biome は _JavaScript_, _TypeScript_, _JSX_ そして _JSON_ 向けの[高速なフォーマッター](./benchmark#formatting)**であり、**[_Prettier_ との互換性は96%](https://console.algora.io/challenges/prettier)** を達成しています。
+
+**Biome は _JavaScript_, _TypeScript_, _JSX_ のための[高性能なリンター](https://github.com/biomejs/biome/tree/main/benchmark#linting)** であり、ESLint, TypeScript ESLint, [その他のソース](https://github.com/biomejs/biome/discussions/3)から **[170以上のルール](https://biomejs.dev/linter/rules/)**を提供しています。Biome は**詳細で文脈に沿った結果を出力**するため、コードを改善し、より良いプログラマになるための手助けをします！
+
+**Biome** は最初から[**エディタ内で対話的に**](https://biomejs.dev/ja/guides/integrate-in-editor/)使用するように設計されています。
+あなたがコードを書いているときに、形の崩れたコードを format と lint することができます。
```
Comment (unvalley) [inline packages/@biomejs/biome/README.ja.md:41]:
```suggestion
**Biome** はウェブプロジェクトのための高性能なツールチェーンであり、プロジェクトの健全性を維持するための開発者ツールの提供を目的としています。

**Biome は _JavaScript_, _TypeScript_, _JSX_ そして _JSON_ 向けの[高速なFormatter](./benchmark#formatting)**であり、**[_Prettier_ との互換性は96%](https://console.algora.io/challenges/prettier)** を達成しています。

**Biome は _JavaScript_, _TypeScript_, _JSX_ のための[高性能なLinter](https://github.com/biomejs/biome/tree/main/benchmark#linting)** であり、ESLint, TypeScript ESLint, [その他のソース](https://github.com/biomejs/biome/discussions/3)から **[170以上のルール](https://biomejs.dev/linter/rules/)**を提供しています。Biome は**詳細で文脈に沿った結果を出力**するため、コードを改善し、より良いプログラマになるための手助けをします！

**Biome** は最初から[**エディタ内で対話的に**](https://biomejs.dev/ja/guides/integrate-in-editor/)使用できるように設計されています。
あなたがコードを書いているときに、形の崩れたコードを format と lint することができます。
```

File: packages/@biomejs/biome/README.ja.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+## Biome をもっと詳しく
+
+**Biome** は理にかなったデフォルト設定を持ち、設定を必要としません。
+
+**Biome** はモダンなウェブ開発における[全ての主要な言語][language-support]をサポートすることを目指しています。
+
+**Biome** は動作するために Node.js を必要としません。
+
+**Biome** はソーステキストを完全に表現し、最高のエラー回復を提供する洗練されたパーサーを持つ、最高級の LSP サポートを提供します。
```
Comment (unvalley) [inline packages/@biomejs/biome/README.ja.md:79]:
```suggestion
Biomeは、ソーステキストの完全な表現力とエラー回復能力を持つ洗練されたParserによって、優れたLSPサポートを提供します。
```

File: packages/@biomejs/biome/README.ja.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+**Biome** は以前は別々のツールだった機能を統合します。共通基盤を構築することで、コードの処理、エラーの表示、並列処理、キャッシュ、設定について統一的な体験を提供することができます。
```
Comment (unvalley) [inline packages/@biomejs/biome/README.ja.md:81]:
```suggestion
**Biome** は、以前は別々のツールで提供されている機能を統合します。共通基盤を構築することで、コードの処理、エラーの表示、並列処理、キャッシュ、設定について統一的な体験を提供します。
```

File: packages/@biomejs/biome/README.ja.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+私たちの[プロジェクトの哲学][biome-philosophy]について詳しく読んでください。
```
Comment (unvalley) [inline packages/@biomejs/biome/README.ja.md:83]:
```suggestion
興味のある方は、[プロジェクトの理念][biome-philosophy]もご覧ください。
```

File: packages/@biomejs/biome/README.ja.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+npx @biomejs/biome format --write ./src
+
+# ファイルをlintする
+npx @biomejs/biome lint ./src
+
+# format、lintなどを実行し、安全な提案を適用する
+npx @biomejs/biome check --apply ./src
+
+# CI環境では、すべてのファイルをformatやlintなどに照らしてチェックする
+npx @biomejs/biome ci ./src
```
Comment (unvalley) [inline packages/@biomejs/biome/README.ja.md:62]:
```suggestion
# CI環境では、すべてのファイルを対象にformatやlintをチェックする
npx @biomejs/biome ci ./src
```

**Review by unvalley — CHANGES_REQUESTED** [review]
Thank you for translation, I left some suggestions!

**Comment by ImBIOS** [comment]
I recommend for translation links, don't use inline links like this: [lang a](url) | [lang b](url) | [lang c](url)

It's prone to merge conflict if in the future we need to translate into different languages at the same time.

I suggest to use  one line per language:

- [lang a](url)
- [lang b](url)
- [lang c](url)

In this way, diff for each line counted as different work, so the probability of merge conflict would be less.

**Comment by ematipico** [comment]
Thank you @ImBIOS , do you agree with me that what you suggested can be done in a different pr?

If you do, would you like to send a PR for fixing that?

**Comment by ImBIOS** [comment]
@ematipico I ultimately agree, let me hunt down this 👾 minion task, 🚚  I'll PR to this branch.

File: packages/@biomejs/biome/README.ja.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+
+**Biome** は理にかなったデフォルト設定を持ち、設定を必要としません。
+
+**Biome** はモダンなウェブ開発における[全ての主要な言語][language-support]をサポートすることを目指しています。
+
+**Biome** は動作するために Node.js を必要としません。
+
+**Biome** はソーステキストの完全な表現力とエラー回復能力を持つ洗練されたParserによって、優れたLSPサポートを提供します。
+
+**Biome** は、以前は別々のツールで提供されていた機能を統合します。共通基盤を構築することで、コードの処理、エラーの表示、並列処理、キャッシュ、設定について統一的な体験を提供します。
```
Comment (unvalley) [inline packages/@biomejs/biome/README.ja.md:75]:
```suggestion
**Biome** は以前は別々のツールで提供されていた機能を統合します。共通基盤を構築することで、コードの処理、エラーの表示、並列処理、キャッシュ、設定について統一的な体験を提供します。
```

File: packages/@biomejs/biome/README.ja.md [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+[npm-badge]: https://badgen.net/npm/v/@biomejs/biome?icon=npm&color=green&label=%40biomejs%2Fbiome
+[npm-url]: https://www.npmjs.com/package/@biomejs/biome/v/latest
+[vscode-badge]: https://badgen.net/vs-marketplace/v/biomejs.biome?label=vscode&icon=visualstudio&color=green
+[vscode-url]: https://marketplace.visualstudio.com/items?itemName=biomejs.biome
+[open-vsx-badge]: https://badgen.net/open-vsx/version/biomejs/biome?label=open-vsx&color=green
+[open-vsx-url]: https://open-vsx.org/extension/biomejs/biome
+
+</div>
+
+**Biome** はウェブプロジェクトのための高性能なツールチェーンであり、プロジェクトの健全性を維持するための開発者ツールの提供を目的としています。
```
Comment (unvalley) [inline packages/@biomejs/biome/README.ja.md:28]:
```suggestion
**Biome** はWebプロジェクトのための高性能なツールチェーンであり、プロジェクトの健全性を維持するための開発者ツールの提供を目的としています。
```

**Review by unvalley — APPROVED** [review]
I've changed a bit, and it looks good for me.

**Comment by K-tecchan** [comment]
thanks!

**Pull request merged**